examples/employee-app/resources/DbCreds.properties

```diff
@@ -0,0 +1,5 @@
+# Oracle DB Credentials

```

Comment:
This file is missing the proper copyright header.

Reply:
We removed this file.